ts/src/program/context.ts

```diff
@@ -33,6 +33,12 @@ export type Context<A extends Accounts = Accounts> = {
    */
   instructions?: TransactionInstruction[];
 
+  /**
+   * Instructions to run *after* a given method. Often this is used, for
+   * example to close accounts after to executing a method.
+   */
+  postInstructions?: TransactionInstruction[];
```

Comment:
Want to add `preInstructions` for symmetry and a test?

Reply:
Added preInstructions 
I only see 1 test file in the repo. Is there some idl mocking value to use for testing ?

Reply:
End to end tests are the easiest thing to do. Can we add a CHANGELOG entry as well.

Reply:
Added 3 tests